src/Chunk.ts

```diff
@@ -712,13 +710,18 @@ export default class Chunk {
 			});
 		}
 
+		if (!this.id) {
```

Comment:
Not a big fan of internal errors that are impossible to test, but there are other opinions out there as well. The price of course are non-null assertions. In a future version of Rollup, maybe we can move away from classes towards data types that evolve as the bundle evolves.

Reply:
Yeh I had that initially but noticed a few more internal errors. Happy to remove if you want

>maybe we can move away from classes towards data types that evolve as the bundle evolves.

sounds good